app/models/application_form.rb

```diff
@@ -3,6 +3,8 @@
 class ApplicationForm < ApplicationRecord
   belongs_to :candidate
   has_many :application_choices
+  has_many :application_work_experiences
+  has_many :application_volunteering_experiences
 
```

Comment:
Would you ever need/want to get all of the experiences for a given application? e.g. `application_form.application_experiences`? Not sure what is really needed here but another way to do this would be to have a single `has_many :application_experiences` association and scopes for `work` and `volunteering`. That way you could view 'experiences' as a single list as well as just work and just volunteering.

Reply:
In the API we're planning to split into separate "jobs" and "volunteering" lists, and it seems to be the same in candidate and provider prototypes, so I'm in favour of keeping them separate (for now)

Reply:
I am also in favour of keeping them separate 👍